### DISCUSSION TITLE
Installing jupytr on the new MacBook Air

### DISCUSSION THREAD

**Discussion by shruti976** [Homebrew/discussions#712] [Everyday usage]
Hi, I am trying to install jupytr using home-brew, on my new Mac Air laptop with M1 chip but its showing me the following error:
ghc@8.8: The x86_64 architecture is required for this software.
Error: An unsatisfied requirement failed this build.
shrutisingh@Shrutis-MacBook-Air ~ % curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh

I am uploading the screenshot of the error as well.
Can anyone help me in downloading jupytr?

<img width="1220" alt="Screen Shot 2021-02-05 at 3 01 41 PM" src="https://user-images.githubusercontent.com/54175576/107083597-e13bac00-67c3-11eb-9afa-b6febd265c16.png">

**Comment by carlocab** [comment]
Do you mean `jupyterlab`? That depends on `pandoc`, which doesn't work on M1 (yet).

That's why you see the error `An unsatisfied requirement failed this build`.

Also, when sharing the output of commands, please avoid using screenshots. They're often difficult to read from and impossible to reuse (e.g. if someone would like to Google an error message).

**Reply by shruti976** [reply]
OH, thanks for the reply, so much, I'll avoid using screenshots, I'm new to GitHub. And yes I meant jupytrlab. What do you mean by pandoc ? Is there another way of installing jupytr? I need it for my course this semester.

**Reply by SMillerDev** [reply]
Pandoc is one of the dependencies but it doesn't work on ARM based devices. 
https://stackoverflow.com/a/64883440

This could help you install a Rosetta based homebrew next to your normal homebrew for cases like this.
